app/commands/thredded/autofollow_users.rb

```diff
@@ -1,18 +1,23 @@
 # frozen_string_literal: true
 module Thredded
-  class AutofollowMentionedUsers
+  class AutofollowUsers
     def initialize(post)
       @post = post
     end
 
     def run
       autofollowers.each do |user|
-        Thredded::UserTopicFollow.create_unless_exists(user.id, post.postable_id, :mentioned)
+        reason = mentioned_users.include?(user) ? :mentioned : :auto
```

Comment:
Perhaps `autofollowers` should return a Hash of `user_id` to `reason` so we don't have to do this check here?